extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -1,5 +1,5 @@
 amp-story-shopping-tag {
-  font-family: 'Poppins', sans-serif !important;
+  font-family: 'Poppins','Arial', sans-serif !important;
```

Comment:
Please revert this line and default to `sans-serif` to follow the [font stack](https://github.com/ampproject/amphtml/blob/main/extensions/amp-story-dev-tools/0.1/amp-story-dev-tools.css#L11) pattern of our codebase.

Updating the font stack might be a good idea but that would be a global change for a separate PR.

Reply:
'sans-serif' is just a font modifier right, not an actual font.
So I'm not sure what it would fall back to If I don't add in something else like Arial here.

Was adding it per Gabriel's request here:
https://github.com/ampproject/amphtml/pull/37188#discussion_r768158694

Anyway, I removed it - we can take care of it in another PR if we need it.